torch/_dynamo/variables/builtin.py

```diff
@@ -1101,6 +1107,21 @@ def call_getattr(
                 unimplemented("tensor grad")
             else:
                 unimplemented("tensor grad")
+        elif name == "__bases__":
+            value = obj.as_python_constant()
+            if isinstance(value, type):
+                bases = value.__bases__
+                if source is None:
+                    assert len(bases) == 1 and (
+                        bases[0] is object or bases[0] is torch._C._TensorBase
+                    ), f"{value.__name__}.bases: {bases}"
+                    tuple_args = [BuiltinVariable(bases[0])]
+                else:
+                    tuple_args = [
+                        VariableBuilder(tx, GetItemSource(source, i))(b)
```

Comment:
Your source here is skipping the `__bases__` access, which, I am not sure is what you want?

Reply:
The `AttrSource` is being created before is if-else chain.